scss/crux.scss

```diff
@@ -22,7 +22,7 @@ mark {
 }
 
 .rbt-input-main {
-  width: 115px !important;
+  min-width: 160px !important;
```

Comment:
Added this, because (list shown in typeahead component has its own style of min-width 160px)